org.filezillaproject.Filezilla.appdata.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2014 Ryan Lerch <rlerch@redhat.com> -->
 <!-- Copyright 2016 Tim Kosse <tim.kosse@filezilla-project.org> -->
-<application>
-  <id type="desktop">org.filezillaproject.Filezilla.desktop</id>
+<component type="desktop">
+  <id>org.filezillaproject.Filezilla.desktop</id>
```

Comment:
```suggestion
  <id>org.filezillaproject.Filezilla</id>
```

And add `launchable`

Reply:
This is in the provides section currently. https://github.com/flathub/org.filezillaproject.Filezilla/pull/28/files#diff-cbb9610615681de4b45d930971da7381R94 shouldn't be in both certainly.

Reply:
No `org.filezilla-project.FileZilla` != `org.filezillaproject.Filezilla`. Remember upstream named it different than here so both are just screwed.